appsmithorg/appsmith
# chore: [WIP] Add failFastApiCalls test

## Description

Adding test for `failFastApiCalls` saga
1. Need to verify is all `triggerEffects` are triggered
2. verify that it waits for all success effects 
3. verify it returns as soon as any of the error effect is triggered

Fixes # (issue)

## Type of change

Add test

## How Has This Been Tested?
 

## Checklist:
